code/modules/mob/mob.dm

```diff
@@ -584,11 +584,20 @@
 /mob/proc/facedir(ndir)
 	if(!canface())
 		return FALSE
+	if(facing_dir)
+		facing_dir = ndir
 	setDir(ndir)
 	if(buckled && !buckled.anchored)
 		buckled.setDir(ndir)
 	return TRUE
 
+/mob/setDir(newdir)
+	if(facing_dir)
+		if(!canface())
+			facing_dir = null
+		else if(newdir != facing_dir)
+			return
+	..()
```

Comment:
what the fuck is thiiiiis

Reply:
stops user from changing their direction when facing_dir is set, used by scopes

Reply:
As discussed in discord, change this